### DISCUSSION TITLE
stage CCPA, MRMS and NDAS for the vx task at Derecho

### DISCUSSION THREAD

**Discussion by SarahLu-NOAA** [ufs-community/ufs-srweather-app#986] [Q&A] [answered]
I'd like to add the vx task (running at Derecho).   The UFS documentation provides the locations to fetch CCPA, MRMS, and NDAS data.  However, these are only for most recent 2 days (for MRMS) and 8 days (for CCPA and NDAS).  The out-of-the-box case for SRW v2.2.0 is for 2019-06-15.    Please suggest where to fetch these retro analysis.  Thanks

**Comment by MichaelLueken** [comment, marked as answer]
Hello, @SarahLu-NOAA,

In order to utilize the staged verification data on Derecho for the out-of-the box case of 2019-06-15, you will need to replace:
```
  CCPA_OBS_DIR: ""
  MRMS_OBS_DIR: ""
  NDAS_OBS_DIR: ""
```
with
```
  CCPA_OBS_DIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/obs_data/ccpa/proc
  MRMS_OBS_DIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/obs_data/mrms/proc
  NDAS_OBS_DIR: /glade/work/epicufsrt/contrib/UFS_SRW_data/v2p2/obs_data/ndas/proc
```
in your config.yaml file.  Doing so will allow you to pull the staged verification from disk.

**Reply by SarahLu-NOAA** [reply]
Thanks, @MichaelLueken, for the info on where staged verification data is located at Derecho.   config.yaml is modified and the exp with vx task runs smoothly.
